src/yoroi-wallets/cardano/ShelleyWallet.ts

```diff
@@ -432,14 +432,14 @@ export class ShelleyWallet implements WalletInterface {
     return changeAddress
   }
 
-  private async getAddressedChangeAddress(): Promise<{address: string; addressing: CardanoTypes.Addressing}> {
+  private getAddressedChangeAddress(): Promise<{address: string; addressing: CardanoTypes.Addressing}> {
     const changeAddr = this.getChangeAddress()
     const addressing = this.getAddressing(changeAddr)
```

Comment:
`getChangeAddress` and `getAddressing` must not throw, it will need to be wrapped within a promise, otherwise `.catch()` won't catch.

Reply:
if there were any `.catch`'s, typescript would have errored?